polish-adblock-filters/adblock.txt

```diff
@@ -3495,15 +3496,15 @@ zywiecinfo.pl##.promoted
 ||static.gazetaprawna.pl/dziennik/required/build/js/adexonLoader.js$script
 ||static.gazetaprawna.pl/dziennik/required/build/js/scrollRedirect.js$script,domain=dziennik.pl
 ||static.im-g.pl/static/content/front/content-generator/master/front/dist/importFrame-min.js
-||static*jelonka.com
+||static*jelonka.com^
```

Comment:
A nie `.` zamiast `*` ?

Reply:
Ja tam nie wiem co to jest, ale raczej jak blokada domen/skryptów to by się przydał daszek.

Dodatkowo te znaki są daleko i by trzeba nie lada wyzwania.

Reply:
> krystian3w : https://github.com/MajkiIT/polish-ads-filter/pull/13334#discussion_r283948243 :

> Ja tam nie wiem co to jest,

Ja też nie, jakiś dziwoląg chyba.

> ale raczej jak blokada domen/skryptów to by się przydał daszek.

Nie pisałem nic o daszku, tylko o `*` , więc nie wiem po co o nim w ogóle wspominasz.

> Dodatkowo te znaki są daleko i by trzeba nie lada wyzwania.

Które znaki, bo nie mam zielonego pojęcia, o które chodzi.